.github/workflows/lint.yml

```diff
@@ -127,6 +127,19 @@ jobs:
           components: rustfmt
       - run: cargo fmt --all --check
 
+  udeps:
+    name: udeps
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@nightly
+      - uses: Swatinem/rust-cache@v2
+        with:
+          cache-on-failure: true
+      - run: cargo install cargo-udeps --locked
+      - run: cargo udeps --workspace --lib --examples --tests --benches --all-features --locked
```

Comment:
Unfortunately it doesn't look like there is an action for udeps. Another solution if this isn't acceptable would be to download the binary from Github releases.

Reply:
Use stable rust and cargo-binstall should work to not have to compile it

Reply:
I see

Reply:
I haven't used cargo-binstall before, does this still work on nightly?